.cirrus.yml

```diff
@@ -170,6 +333,120 @@ task:
+
+task:
+  only_if: $CIRRUS_PR != ''
+
+  windows_container:
+    dockerfile: .ci-dockerfiles/windowsservercore/Dockerfile
+    os_version: 2019
+    cpu: 8
+    memory: 24
+
+  name: "PR: x86-64-pc-windows-msvc"
+
+  libs_cache:
+    folder: build/libs
+    fingerprint_script:
+      - ps: (Get-FileHash -Path lib\CMakeLists.txt).Hash
+    populate_script:
+      - ps: .\make.ps1 -Command libs -Generator "Visual Studio 16 2019"
+
+  config_script:
+    - ps: .\make.ps1 -Command configure -Config Release -Generator "Visual Studio 16 2019" -Prefix "build\install\release"
+  build_script:
+    - ps: .\make.ps1 -Command build -Config Release -Generator "Visual Studio 16 2019" -Prefix "build\install\release"
+  test_script:
+    - ps: .\make.ps1 -Command test -Config Release -Generator "Visual Studio 16 2019" -Prefix "build\install\release"
+
+task:
+  only_if: $CIRRUS_PR != ''
+
+  container:
+    image: ponylang/ponyc-ci-cross-arm:20200127
+    cpu: 8
+    memory: 24
+
+  name: "PR: cross-compile: arm-unknown-linux-gnueabi"
+
+  libs_cache:
+    folder: build/libs
+    fingerprint_script: echo "`md5sum lib/CMakeLists.txt` cross arm"
+    populate_script: make libs build_flags=-j8 llvm_archs="X86;ARM"
+
+  configure_script:
+    - make configure
+  build_script:
+    - make build build_flags=-j8
+  test_script:
+    - make test-ci
+  libponyrt_cross_script:
+    - make cross-libponyrt CC=arm-linux-gnueabi-gcc CXX=arm-linux-gnueabi-g++ arch=armv7-a tune=cortex-a9 llc_arch=arm
+  test_stdlib_cross_script:
+    - make test-cross-ci PONYPATH=../armv7-a/release cross_triple=arm-unknown-linux-gnueabi cross_arch=armv7-a cross_cpu=cortex-a9 cross_linker=arm-linux-gnueabi-gcc cross_runner="qemu-arm-static -cpu cortex-a9 -L /usr/local/arm-linux-gnueabi/libc"
+
+task:
+  only_if: $CIRRUS_PR != ''
+
+  container:
+    image: ponylang/ponyc-ci-cross-armhf:20200212
+    cpu: 8
+    memory: 24
+
+  name: "PR: cross-compile: arm-linux-gnueabihf-gcc"
+
+  libs_cache:
+    folder: build/libs
+    fingerprint_script: echo "`md5sum lib/CMakeLists.txt` cross arm"
+    populate_script: make libs build_flags=-j8 llvm_archs="X86;ARM"
+
+  configure_script:
+    - make configure
+  build_script:
+    - make build build_flags=-j8
   test_script:
-    - make -f Makefile-lib-llvm default_pic=true arch=x86-64 config=release -j8
-    - make -f Makefile-lib-llvm default_pic=true arch=x86-64 config=release test-ci
+    - make test-ci
+  libponyrt_cross_script:
+    - make cross-libponyrt CC=arm-linux-gnueabihf-gcc CXX=arm-linux-gnueabihf-g++ arch=armv7-a tune=cortex-a9 llc_arch=arm
+  test_stdlib_cross_script:
+    - make test-cross-ci PONYPATH=../armv7-a/release cross_triple=arm-unknown-linux-gnueabihf cross_arch=armv7-a cross_cpu=cortex-a9 cross_linker=arm-linux-gnueabihf-gcc cross_runner="qemu-arm-static -cpu cortex-a9 -L /usr/local/arm-linux-gnueabihf/libc"
+
+# AArch64 has been broken for some time
```

Comment:
seems like we should just delete all the commented out stuff, sound reasonable?

Reply:
Sure.  Perhaps we should add an issue to take another look at AArch64?

Reply:
sounds like an excellent idea. open that issue!